### PR TITLE
Add a polyfill to ds for Node < 12

### PR DESCRIPTION
**Pull request by flxwu**

### CODE REVIEW

**Comment by viktor-ku** [comment]
Won't be needed after this PR https://github.com/entropic-dev/entropic/pull/186 since saving/loading configs is done in sync mode there

**Comment by zacanger** [comment]
Even with the changes in #186, Node < 12 would have issues. If the goal is to support down that many versions, it might make sense to bring in `@babel/polyfill` or something like it. I know I've at least seen `[].flat` somewhere, which would be an error.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The only thing about Entropic that assumes you're managing javascript packages i
 
 ### The `ds` cli
 
-Entropic requires a new command-line client, called `ds` (or "entropy delta".) **`ds` requires at least Node 12.** Install the cli:
+Entropic requires a new command-line client, called `ds` (or "entropy delta".) **`ds` requires at least Node 8.9** Install the cli:
```
[the remark this answers is not in the document]
Comment (flxwu) [reply README.md]: 👍

**Comment by chrisdickinson** [comment]
(Merge queue notes: update the readme)

**Comment by chrisdickinson** [comment]
Thanks for your patience! Merged in https://github.com/entropic-dev/entropic/commit/05af6fd7d71d55ec6f97b36b47468df1eb348453!

**Pull request closed without merging**